.github/workflows/build.yml

```
@@ -29,65 +35,23 @@ jobs:
 
   # Publishing is done in a separate job to allow
   # for all matrix builds to complete.
-  release:
+  BuildRelease:
     needs: test
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/main'
+    name: Checkout Code
     steps:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
           node-version: 14.x
-
       - name: Check out repo
         uses: actions/checkout@v2
         with:
           fetch-depth: 2
```

```suggestion
          fetch-depth: 0
```
https://github.com/JupiterOne/integration-template/pull/112#discussion_r1151083166